hc-bar/js/bar.js

```diff
@@ -0,0 +1,314 @@
+// series is an array of objects
+//    name - eventType (Intercept, Strike, Other)
+//    data is an array of objects
+//      name - monthYear
+//      Y - # of event in m/y
+//      drilldown - monthYear-eventType
+
+// drilldown is an object
+//    allowPointDrilldown - false (drilldown shows events for all categories in that month/year)
+//    series is an array of objects
+//      name - eventType+Headline+Text (Strike, Intercept, Other)
+//      id - monthYear-eventType
+//      data is an array of arrays
+//        [day, # of events]
+
+var drilldownSeries = [];
+// Create empty array for new event objects
+var series = [];
+
+Highcharts.data({
+  // Load Data in from Google Sheets
+  googleSpreadsheetKey: "1AKM59m3iaOSSQgVsIoDcSWxjcpX781JIuBwB9YdDgFs",
+  googleSpreadsheetWorksheet: 1,
+  switchRowsAndColumns: true,
+  parsed: function parsed(columns) {
+    // set default values
+    var drilldown = "";
+    var dataObject = { other: {}, strike: {}, intercept: {} };
+    var dataArray = [];
+    var drilldownObject = {};
+
+    columns.forEach(function (row, index) {
+      // Skip header row and title row
+      if (index == 0 || index == 1) {
+        return;
+      }
+      // Get event value for this row
+      var interceptColored = row[10].toLowerCase().indexOf("yellowgreen") > -1;
+      var strikeColored = row[10].toLowerCase().indexOf("indianred") > -1;
+      if (interceptColored) {
+        eventRow = "intercept";
+      } else if (strikeColored) {
+        eventRow = "strike";
+      } else {
+        eventRow = "other";
+      }
+      // Remove links from the text that will appear in the tooltip
+      var eventText = row[10].substr(0, row[10].indexOf("<a "));
```

Comment:
Lines 39-47

Normally pulling code out into variables helps clean things up and make code more readable. In this excerpt however, the use of colors (indianred/interceptColored) and strikeColored makes the code block a bit more difficult to understand at first glance. 

One way to clean this up is by removing the abstraction that is provided by the colors and stick to just one variable type. Since you are setting the evenrow to strike, intercept and other here - try and use that: 

if (row[10].indexOf(">STRIKE<") > -1) {
     eventRow = "strike"
} else if (row[10].indexOf(">INTERCEPT<") > -1) {
     eventRow = "intercept"
} else {
     eventRow = "other"
}

The carrots are around STRIKE/INTERCEPT to prevent any other terms to be passed in from the google sheet.